pkg/cluster/node.go

```diff
@@ -108,7 +108,6 @@ func GetNodeIP(client client.Client) (string, error) {
 		log.Error(err, "Getting NodeIP From kube-api-server")
 		os.Exit(1)
 	}
-
 	nodeIP := nodeIPs[0]
```

Comment:
we can add for len(nodeIPs) > 0 before doing this

Reply:
@gourishkb added similar check on line 107,checking if len is 0